plugins/Diagnostics/lang/en.json

```diff
@@ -6,6 +6,7 @@
         "MysqlMaxPacketSize": "Max Packet Size",
         "MysqlMaxPacketSizeWarning": "It is recommended to configure a 'max_allowed_packet' size in your MySQL database of at least %1$s. Configured is currently %2$s.",
         "ConfigFileIntroduction": "Here you can view the Matomo configuration. If you are running Matomo in a load balanced environment, the page might be different depending on from which server this page is loaded. Rows with a different background color are changed config values that are specified for example in the %1$s file.",
+        "EnableRequiredDirectoriesDiagnostic": "This check was skipped as this check is disabled in the config. To enable this check set [General] enable_required_directories_diagnostic in the \"config/config.ini.php\" file.",
```

Comment:
Just tested it and worked generally 👍  The last tweak I would make is to change the part `[General] enable_required_directories_diagnostic in the` to `[General] enable_required_directories_diagnostic = 1 in the` so it's more clear what needs to happen as otherwise users may be confused. Then it should be good to merge.